### ISSUE TITLE
Displacy jupyter rendering issue on one entity

### ISSUE THREAD

**Issue by vivekam101** [explosion/spaCy#2630]
im trying recognize ner with a single entity "chemical" and would like to see each classified entities seperately using displacy in jupyter but not happening. Please check screenshot. Thanks

![screenshot from 2018-08-04 14-09-15](https://user-images.githubusercontent.com/5806230/43674345-30a73cd8-97f0-11e8-8524-92f60521eacd.png)

**Comment by ines** [comment]
What are the actual entities recognized in the text? Could you run the following:

```python
for ent in doc.ents:
    print(ent.text, ent.label_, ent.start_char, ent.end_char)
```

And which version of spaCy are you using? You can check this by running the following command (which will give you all relevant details as copy-pastable Markdown):

```bash
python -m spacy info --markdown
```

**Comment by vivekam101** [comment]
## Info about spaCy

* **Python version:** 3.5.5
* **Models:** en
* **spaCy version:** 2.0.11
* **Platform:** Linux-4.15.0-29-generic-x86_64-with-debian-buster-sid

![screenshot from 2018-08-04 15-37-16](https://user-images.githubusercontent.com/5806230/43675110-523d5cb8-97fc-11e8-92f2-172862253f6a.png)

training set size is of 2500 records extracted from wiki. i have trained with dropout rate dropout_from=0.5 dropout_to=0.40 dropout_decay=.01 spacy-train.  Will that be the reason ?

**Comment by ines** [comment]
Yeah, it looks like the problem here is that the entity you see highlighted in the visualizer is the *actual* entity span predicted by your model. So your model is predicting those very long spans of text.

What does you training data look like? Is it possible that there was some conversion mistake that caused the entity boundaries to not be set correctly?

**Comment by vivekam101** [comment]
Yes Ines, problem was with training. Fixed it. Closing the bug. Thanks
